src/tree/registries/UnifiedRegistryTreeDataProvider.ts

```diff
@@ -0,0 +1,132 @@
+import { RegistryDataProvider } from '@microsoft/vscode-docker-registries';
+import * as vscode from 'vscode';
+
+export interface UnifiedRegistryItem<T> {
+    provider: RegistryDataProvider<T>;
+    wrappedItem: T;
+    parent: UnifiedRegistryItem<T> | undefined;
+}
+
+// eslint-disable-next-line @typescript-eslint/naming-convention

```

Comment:
This linter suppression may be able to be removed; the rules are different in the Docker extension repo